lib/index.js

```diff
@@ -308,12 +310,44 @@ class NewsArticle extends BaseAsset {
             }
         }
 
+        let quote_svg_path = path.join(__dirname, './assets/quote.svg');
+        let quote_decoration_xml = fs.readFileSync(quote_svg_path, 'utf8');
+        let quote_decoration = cheerio.load(quote_decoration_xml, {
+            xmlMode: true,
+        });
+        cheerio('blockquote', this._body).prepend(`<div class="quote-decoration">
+            ${quote_decoration.html('svg')}
+        </div>`);
```

Comment:
I'm a little confused by this -are we assuming that everything is surrounded by quotes?